q2_phylogeny/plugin_setup.py

```diff
@@ -418,6 +422,41 @@
                  "are not tip identifiers in tree.")
 )
 
+T2 = TypeMatch([Rooted, Unrooted])
+
+T3 = (Frequency | RelativeFrequency | PresenceAbsence | Composition)
+
+plugin.methods.register_function(
+    function=q2_phylogeny.filter_tree,
+    inputs={'tree': Phylogeny[T2],
+            'table': FeatureTable[T3],
```

Comment:
Shouldn't the table be listed under parameters, as it is optional? Or is this a type-map requirement?